### ISSUE TITLE
How to perform simple transfer and query between nodes in mixin setuptestnet？

### ISSUE THREAD

**Issue by ShallMate** [MixinNetwork/mixin#80]
How can I transfer funds to the nodes in the set testnet? How can I check the transfer result after the transfer? I think it should be in the document, but I think many places are described very vaguely. I don't know which two rpc interfaces to call for transfer and query

**Comment by cedricfung** [comment]
Yes, the doc is missing. You may try this repo https://github.com/learnforpractice/mixin-python

**Comment by ShallMate** [comment]
@cedricfung thanks very much，I will try it

**Comment by ShallMate** [comment]
@cedricfung  Hello, I used the repo you provided. But I can't compile it, the reason seems to be because the ubuntu system can't compile. I got the following error:
********************************************************************************
scikit-build could not get a working generator for your system. Aborting build.

Building Linux wheels for Python 3.6 requires a compiler (e.g gcc).
But scikit-build does *NOT* know how to install it on ubuntu

To build compliant wheels, consider using the manylinux system described in PEP-513.
Get it with "dockcross/manylinux-x64" docker image:

  https://github.com/dockcross/dockcross#readme

For more details, please refer to scikit-build documentation:

  http://scikit-build.readthedocs.io/en/latest/generators.html#linux

********************************************************************************
If I operate on the setuptestnet under this warehouse, how can I make transfers and inquiries?

**Comment by ShallMate** [comment]
@cedricfung If I operate on the setuptestnet under this repo, how can I perform the transfer and check the transfer result?

**Comment by cedricfung** [comment]
No docs yet. But you can get all things from rpc/consensus_test.go

**Comment by learnforpractice** [comment]
> @cedricfung Hello, I used the repo you provided. But I can't compile it, the reason seems to be because the ubuntu system can't compile. I got the following error:
> 
> scikit-build could not get a working generator for your system. Aborting build.
> 
> Building Linux wheels for Python 3.6 requires a compiler (e.g gcc).
> But scikit-build does _NOT_ know how to install it on ubuntu
> 
> To build compliant wheels, consider using the manylinux system described in PEP-513.
> Get it with "dockcross/manylinux-x64" docker image:
> 
> https://github.com/dockcross/dockcross#readme
> 
> For more details, please refer to scikit-build documentation:
> 
> http://scikit-build.readthedocs.io/en/latest/generators.html#linux
> 
> If I operate on the setuptestnet under this warehouse, how can I make transfers and inquiries?

To build mixin-python from source code, you need to install clang & go 1.6.1 & cmake first.
Or you can try it in jupyter notebook online:
https://mybinder.org/v2/gh/learnforpractice/mixin-python/HEAD?filepath=notebook%2Fhelloworld.ipynb

**Comment by ShallMate** [comment]
@cedricfung  about test_testnet.py https://prsdigg.com/articles/4375be62-6ef4-4de2-af9a-086816380fde
Sorry to disturb you, I found in the test_testnet.py file that you are based on this blog. However, this blog requires digital currency to be purchased. It is not that I don’t want to pay, but I really don’t have experience in purchasing digital currency. Can you share my content about this blog? I think it can help me solve the problem I am facing now. Many problems. Thank you very much if you can.

**Comment by cedricfung** [comment]
What's your Mixin ID, I can send you some BTC.

**Comment by ShallMate** [comment]
@cedricfung  Thank you so much, I feel embarrassed to bother you so much. My Mixin ID is 39781041. Thank you again.
